src/spec/authentication_service.md

```diff
@@ -230,14 +226,13 @@ Delete the user account with the given user id.
 
 ```rust
 struct DeleteUserParams {
-  user_id: UserId,
   opaque_ke3: OpaqueKe3,
 }
 ```
 
 The AS performs the following actions:
 
-* look up the OPAQUE `server_state` in the ephemeral DB based on the `client_id`
+* look up the OPAQUE `server_state` in the ephemeral DB based on the `client_id` given as part of the client's credential-based authentication
```

Comment:
Is 'given' a synonym for 'assigned'? If so, substitute it. If not, I don't understand what it means.

Reply:
The user account deletion endpoint requires Client 2FA authentication, for which one factor is client credential auth. What I was trying to say here is that the client id used for the look up is meant to be taken from the credential used during authentication.

Reply:
This just sounds unusual, "given" is not really a technical term. Do you mean "look up the OPAQUE `server_state` in the ephemeral DB based on the `client_id` assigned through the client's credential-based authentication"?

The same should also be applied to line 138.